src/networking.c

```diff
@@ -1797,13 +1805,18 @@ int freeClientsInAsyncFreeQueue(void) {
     return freed;
 }
 
+/* Return a client by id or null from a given rax struct. */
+client *lookupClientByIDGeneric(rax* rax, uint64_t id) {
+    id = htonu64(id);
+    client *c = raxFind(rax,(unsigned char*)&id,sizeof(id));
```

Comment:
`raxFind()` had changed, perhaps you could merge unstable.

Reply:
Done